src/plugins/guided_onboarding/public/components/guide_panel.tsx

```diff
@@ -289,7 +289,13 @@ export const GuidePanel = ({ api, application }: GuidePanelProps) => {
           </EuiFlyoutBody>
 
           <EuiFlyoutFooter css={styles.flyoutOverrides.flyoutFooter}>
-            <EuiFlexGroup alignItems="center" justifyContent="center" gutterSize="xs">
+            <EuiFlexGroup
+              alignItems="center"
+              justifyContent="center"
+              gutterSize="xs"
+              responsive={false}
+              wrap={true}
```

Comment:
nit: I don't think we need `wrap: true` here because the items in the footer should not go onto another line.